src/EnergyPlus/FluidCoolers.cc

```diff
@@ -1223,247 +850,219 @@ namespace FluidCoolers {
                     UA0 = 0.0001 * DesFluidCoolerLoad; // Assume deltaT = 10000K (limit)
                     UA1 = DesFluidCoolerLoad;          // Assume deltaT = 1K
-                    SimpleFluidCoolerInlet(FluidCoolerNum).WaterTemp = PlantSizData(PltSizCondNum).ExitTemp + PlantSizData(PltSizCondNum).DeltaT;
-                    SimpleFluidCoolerInlet(FluidCoolerNum).AirTemp = SimpleFluidCooler(FluidCoolerNum).DesignEnteringAirTemp;
-                    SimpleFluidCoolerInlet(FluidCoolerNum).AirWetBulb = SimpleFluidCooler(FluidCoolerNum).DesignEnteringAirWetBulbTemp;
-                    SimpleFluidCoolerInlet(FluidCoolerNum).AirPress = StdBaroPress;
-                    SimpleFluidCoolerInlet(FluidCoolerNum).AirHumRat = PsyWFnTdbTwbPb(SimpleFluidCoolerInlet(FluidCoolerNum).AirTemp,
-                                                                                      SimpleFluidCoolerInlet(FluidCoolerNum).AirWetBulb,
-                                                                                      SimpleFluidCoolerInlet(FluidCoolerNum).AirPress,
-                                                                                      CalledFrom);
-                    SolveRoot(Acc, MaxIte, SolFla, UA, SimpleFluidCoolerUAResidual, UA0, UA1, Par);
+                    this->WaterTemp = DataSizing::PlantSizData(PltSizCondNum).ExitTemp + DataSizing::PlantSizData(PltSizCondNum).DeltaT;
+                    this->AirTemp = this->DesignEnteringAirTemp;
+                    this->AirWetBulb = this->DesignEnteringAirWetBulbTemp;
+                    this->AirPress = DataEnvironment::StdBaroPress;
+                    this->AirHumRat = Psychrometrics::PsyWFnTdbTwbPb(this->AirTemp, this->AirWetBulb, this->AirPress, CalledFrom);
+                    General::SolveRoot(Acc, MaxIte, SolFla, UA, SimpleFluidCoolerUAResidual, UA0, UA1, Par);
                     if (SolFla == -1) {
                         ShowWarningError("Iteration limit exceeded in calculating fluid cooler UA.");
-                        ShowContinueError("Autosizing of fluid cooler UA failed for fluid cooler = " + SimpleFluidCooler(FluidCoolerNum).Name);
-                        ShowContinueError("The final UA value =" + RoundSigDigits(UA, 2) + " W/K, and the simulation continues...");
+                        ShowContinueError("Autosizing of fluid cooler UA failed for fluid cooler = " + this->Name);
+                        ShowContinueError("The final UA value =" + General::RoundSigDigits(UA, 2) + " W/K, and the simulation continues...");
                     } else if (SolFla == -2) {
-                        SimSimpleFluidCooler(int(Par(2)), Par(3), Par(4), UA0, OutWaterTempAtUA0);
-                        SimSimpleFluidCooler(int(Par(2)), Par(3), Par(4), UA1, OutWaterTempAtUA1);
+                        CalcFluidCoolerOutlet(int(Par(2)), Par(3), Par(4), UA0, OutWaterTempAtUA0);
+                        CalcFluidCoolerOutlet(int(Par(2)), Par(3), Par(4), UA1, OutWaterTempAtUA1);
```

Comment:
This is exactly what should happen. Call the Calc routine intermediately and then call Sim at the end to update nodes and reports (was Sim called at the end?).

Reply:
The Sim and Calc functions, along with their naming conventions, were all intertwined and messy.  I just cleaned things up.  For now it should be behaving exactly like it was, no diffs, but ultimately this effort should lead to cleaning up the amount of code into smaller, more efficient, units.